### PR TITLE
[Documentation][zos_apf] Add and standarize docstrings on modules/zos_apf.py

### PR DESCRIPTION
**Pull request by IsaacVRey**
##### SUMMARY

Update docstrings on zos_apf to the numpy standard
This addresses part of #1316 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
zos_apf.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by richp405 — APPROVED** [review]

**Review by fernandofloresg — APPROVED** [review]

**Comment by fernandofloresg** [comment]
Do not merge until #1388 is merged.

**Pull request merged**
